client/tailscale/acl.go

```diff
@@ -13,6 +13,8 @@ import (
 	"fmt"
 	"net/http"
 	"net/netip"
+
+	"tailscale.com/util/httpclient"
```

Comment:
I don't think we need a whole new package for this. `httpclient` as a new package is quite the ambitious name for a single func. :)

Can you just add a unexported "func is2xx" func?

```go
func is2xx(r *http.Response) bool { return 200 <= r.StatusCode && r.StatusCode <= 299 }
```

Reply:
Yep! And agreed. 😊